apps/frontend/src/app/loading.tsx

```diff
@@ -0,0 +1,22 @@
+import { LoadingLogo } from "../lib/icons/Logo"
+
+const loading = () => {
+  return (
+    <div className="flex h-screen items-center justify-center bg-[#101010]">
+      <div className="flex flex-col justify-center items-center gap-12">
+        <div className="relative">
+          <div className="absolute inset-0 rounded-full border-2 border-[#464748] animate-pulsate"></div>
+          <div className="rounded-full p-3 relative z-10">
+            <LoadingLogo />
+          </div>
+        </div>
+        <div className="flex flex-col justify-center items-center gap-5">
+          <p className="text-[#EDEDED]">sit tight, loading some javascripts</p>

```

Comment:
We already have predefined colors in tailwind.config.js, you can use that rather than adding a hex code.
Incase if the colors change in the future, we just have to change it at one place rather than changing it one by one